### PR TITLE
Adds missing pieces to documentation

### PR DESCRIPTION
**Pull request by joaoandremartins**
Service specific credentials not specified in some places. Fixes some
other inconsistencies in docs.

### CODE REVIEW

File: spring-cloud-gcp-docs/src/main/asciidoc/storage.adoc [hunk 1]
```diff
@@ -53,3 +53,20 @@ try (OutputStream os = ((WritableResource) gcsResource).getOutputStream()) {
...
+
+=== Configuration
+
+The Spring Boot Starter for Google Cloud Storage provides the following configuration options:
+
+[source,yaml]
+----
+spring.cloud.gcp.storage.auto-create-files=false #optional, default true
+spring.cloud.gcp.storage.credentials.location=[CREDENTIALS_RESOURCE_PATH] #optional
+spring.cloud.gcp.storage.credentials.scope=[COMMA_DELIMITED_SCOPE_LIST] #optional
```
Comment (meltsufin) [inline spring-cloud-gcp-docs/src/main/asciidoc/storage.adoc]: `scope` -> `scopes`?

File: spring-cloud-gcp-docs/src/main/asciidoc/storage.adoc [hunk 2]
```diff
@@ -53,3 +53,20 @@ try (OutputStream os = ((WritableResource) gcsResource).getOutputStream()) {
...
+----
+
+`spring.cloud.gcp.storage.auto-create-files` creates files on Google Cloud Storage when writes are
+made to non-existent files.
+
+`spring.cloud.gcp.storage.credentials.location` and `spring.cloud.gcp.storage.credentials.scope`
```
Comment (meltsufin) [inline spring-cloud-gcp-docs/src/main/asciidoc/storage.adoc]: scopes?

File: spring-cloud-gcp-docs/src/main/asciidoc/storage.adoc [hunk 3]
```diff
@@ -53,3 +53,20 @@ try (OutputStream os = ((WritableResource) gcsResource).getOutputStream()) {
...
+are overrides to the same options in the <<spring-cloud-gcp-core, Spring Cloud GCP Core Module>>.
```
Comment (artembilan) [inline spring-cloud-gcp-docs/src/main/asciidoc/storage.adoc]: In the `pubsub.adoc` change you don't use space after the comma for the same expression.
I wonder if they are identical for the final link or we have to fix this or that for proper rendering.

**Comment by joaoandremartins** [comment]
This is ready for a new review.

File: spring-cloud-gcp-docs/src/main/asciidoc/trace.adoc [hunk 4]
```diff
@@ -53,13 +53,15 @@ messages exceed its buffer size or have been buffered for longer than its schedu
...
 There are several parameters you can use to tune the Stackdriver Trace adapter. All configurations are optional:
 
-
 |===
 | Property name | Description | Default value
 | `spring.cloud.gcp.trace.enabled` | Auto-configure Spring Cloud Sleuth to send traces to Stackdriver Trace. | `true`
 | `spring.cloud.gcp.trace.executor-threads` | Number of threads to use by the underlying gRPC channel to send the trace request to Stackdriver. | `4`
 | `spring.cloud.gcp.trace.buffer-size-bytes` | Buffer size in bytes. Traces will be flushed to Stackdriver when buffered trace messages exceed this size. | 1% of Runtime.totalMemory()
 | `spring.cloud.gcp.trace.scheduled-delay-seconds` | Buffered trace messages will be flushed to Stackdriver when buffered longer than scheduled delays (in seconds) even if the buffered message size didn't exceed the buffer size. | `10`
+| `spring.cloud.gcp.trace.project-id` | Overrides the project ID from the <<spring-cloud-gcp-core, Spring Cloud GCP Core Module>> |
```
Comment (artembilan) [inline spring-cloud-gcp-docs/src/main/asciidoc/trace.adoc]: Should we remove the space here after comma before ` Spring Cloud` in the link as well?
Maybe we don't need to worry at all?
Would you mind to share screen shot how the final doc looks here?

Thanks

Comment (joaoandremartins) [reply spring-cloud-gcp-docs/src/main/asciidoc/trace.adoc]: screenshot from 2017-10-24 11 30 45

**Pull request merged**